src/components/PlayerView/index.tsx

```diff
@@ -81,6 +81,17 @@ export function PlayerView(props: PlayerViewProps) {
     <NativePlayerView
       ref={nativeView}
       style={style}
+      onAdBreakFinished={proxy(props.onAdBreakFinished)}
+      onAdBreakStarted={proxy(props.onAdBreakStarted)}
+      onAdClicked={proxy(props.onAdClicked)}
+      onAdError={proxy(props.onAdError)}
+      onAdFinished={proxy(props.onAdFinished)}
+      onAdManifestLoad={proxy(props.onAdManifestLoad)}
+      onAdManifestLoaded={proxy(props.onAdManifestLoaded)}
+      onAdQuartile={proxy(props.onAdQuartile)}
+      onAdScheduled={proxy(props.onAdScheduled)}
+      onAdSkipped={proxy(props.onAdSkipped)}
+      onAdStarted={proxy(props.onAdStarted)}
```

Comment:
@pogist Is the order of properties pre-defined here? Or can it be in any order?

Asking because if the order must match this then it would be weird to start with ad break finished first as that is literally the last event that happens during ad playback (except if error happens).

Reply:
No, the order makes no difference. And I kept those properties in alphabetical order just to ease reading/refactoring, but that's not required either.